
You reached the start of the range â†’ May 31, 2025 6:55 PM

Starting Container

    at async ModuleLoader.import (node:internal/modules/esm/loader:337:24)

    at async loadESM (node:internal/process/esm_loader:34:7)

    at async handleMainPromise (node:internal/modules/run_main:106:12) {

  code: 'ERR_INVALID_ARG_TYPE'

}

 

Node.js v18.20.5

Available environment variables: [ 'DATABASE_URL' ]

DATABASE_URL exists: true

node:internal/errors:496

    ErrorCaptureStackTrace(err);

    ^

 

TypeError [ERR_INVALID_ARG_TYPE]: The "paths[0]" argument must be of type string. Received undefined

    at new NodeError (node:internal/errors:405:5)

    at validateString (node:internal/validators:162:11)

    at Object.resolve (node:path:1097:7)

    at file:///app/dist/index.js:1338:18

    at ModuleJob.run (node:internal/modules/esm/module_job:195:25)

    at async ModuleLoader.import (node:internal/modules/esm/loader:337:24)

    at async loadESM (node:internal/process/esm_loader:34:7)

    at async handleMainPromise (node:internal/modules/run_main:106:12) {

  code: 'ERR_INVALID_ARG_TYPE'

}

 

Node.js v18.20.5

Available environment variables: [ 'DATABASE_URL' ]

DATABASE_URL exists: true

node:internal/errors:496

    ErrorCaptureStackTrace(err);

    ^

 

TypeError [ERR_INVALID_ARG_TYPE]: The "paths[0]" argument must be of type string. Received undefined

    at new NodeError (node:internal/errors:405:5)

    at validateString (node:internal/validators:162:11)

    at Object.resolve (node:path:1097:7)

    at file:///app/dist/index.js:1338:18

    at ModuleJob.run (node:internal/modules/esm/module_job:195:25)

    at async ModuleLoader.import (node:internal/modules/esm/loader:337:24)

    at async loadESM (node:internal/process/esm_loader:34:7)

    at async handleMainPromise (node:internal/modules/run_main:106:12) {

  code: 'ERR_INVALID_ARG_TYPE'

}

 

Node.js v18.20.5

Available environment variables: [ 'DATABASE_URL' ]

DATABASE_URL exists: true

node:internal/errors:496

    ErrorCaptureStackTrace(err);

    ^

 

TypeError [ERR_INVALID_ARG_TYPE]: The "paths[0]" argument must be of type string. Received undefined

    at new NodeError (node:internal/errors:405:5)

    at validateString (node:internal/validators:162:11)

    at Object.resolve (node:path:1097:7)

    at file:///app/dist/index.js:1338:18

    at ModuleJob.run (node:internal/modules/esm/module_job:195:25)

node:internal/errors:496

    ErrorCaptureStackTrace(err);
